packages/cli/src/databases/entities/WorkflowEntity.ts

```diff
@@ -90,32 +85,8 @@ export class WorkflowEntity extends AbstractEntity implements IWorkflowDb {
 	})
 	pinData: ISimplifiedPinData;
 
-	/**
-	 * Hash of editable workflow state.
-	 */
+	@Column({ length: 36 })
```

Comment:
We are no longer hashing, so `versionId` instead of `hash`?

Reply:
Good idea, changed!